tests/cypress/tests/05-filter/01-filters.spec.js

```diff
@@ -1,47 +0,0 @@
-/** *****************************************************************************
- * Licensed Materials - Property of Red Hat, Inc.
- * Copyright (c) 2020 Red Hat, Inc.
- ****************************************************************************** */
-
-/// <reference types="cypress" />
-
-import { squad, tags } from '../../config'
-import { searchPage, searchBar } from '../../views/search'
-
-// Set filter registry for the test run.
-const filters = [
-  { type: 'apigroup', skip: false },
-  { type: 'container', skip: false },
-  { type: 'created', skip: false },
-  { type: 'kind', skip: false },
-  { type: 'namespace', skip: false },
-  { type: 'status', skip: false },
-]
```

Comment:
Will need to re-enable the filter test without the kind and namespace filters. This can be handled within a future PR.